it/repositories/DepartureRepositorySpec.scala

```diff
@@ -200,9 +200,13 @@ class DepartureRepositorySpec
 
```

Comment:
can you expand on why this has changed?

Reply:
@RobertLaverick when I ran integration tests I saw that there were failures for the ones I modified due to the datetime comparison being different in precision - I modified the precision for the match to work and the test to pass...happy to revert this change if you don't think it's necessary though